controller/instance_manager_controller.go

```diff
@@ -412,15 +405,30 @@ func (imc *InstanceManagerController) syncStatusWithPod(im *longhorn.InstanceMan
 		} else {
 			im.Status.CurrentState = longhorn.InstanceManagerStateStarting
 		}
```

Comment:
Why we remove the below code? If the pod is unknown, and it also means it could be still running somehow. 

```go
	case corev1.PodUnknown:
		im.Status.CurrentState = longhorn.InstanceManagerStateUnknown
```

Reply:
Same question. Other parts LGTM.

Reply:
logic moved into `syncStatusWithNode(im)`?

Reply:
It was already deprecated a long time ago:
```
// Deprecated: It isn't being set since 2015 (74da3b14b0c0f658b3bb8d2def5094686d0e9095)
```

Reply:
Oh then I am ok with this.